invenio/base/config.py

```diff
@@ -75,24 +75,29 @@
     'invenio.legacy.websubmit',
 ]
 
-CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
+_CFG_PREFIX = distutils.sysconfig.get_config_var("prefix")
```

Comment:
We might also add list of deprecated config keys in `invenio/ext/confighacks.py`.